core/drivers/clk/sam/sub.mk

```diff
@@ -5,4 +5,4 @@ at91-common += at91_utmi.c at91_master.c at91_h32mx.c at91_usb.c
 at91-common += at91_programmable.c at91_system.c at91_peripheral.c
 at91-common += at91_generated.c at91_i2s_mux.c at91_audio_pll.c
 
-srcs-$(CFG_SAMA5D2_CLK) += $(at91-common) sama5d2_clk.c
+srcs-$(CFG_DRIVERS_SAMA5D2_CLK) += $(at91-common) sama5d2_clk.c
```

Comment:
Why not use this CFG name in the parent commit? ("drivers: add sub.mk for sam")

Reply:
That's an error, I made a fixup on the wrong commit. This one should be in the previous commit. I used `CFG_DRIVERS_SAMA5D2_CLK` since I think it more coherent with the other CFG names, tell me if you want to keep `CFG_SAMA5D2_CLK` for some reason.